Tech-Lab-On-Campus/Resources/Python-Basics.md

```diff
@@ -0,0 +1,55 @@
+# Python Basics
+
+### Creating a Class
+Classes are the building blocks of object-oriented programming. In python, to create a class
+we use the keyword `class` to define a class. For example, let's say I want to create a class called Animal. Within a class, there is a method called `__init__` which is an instance method that initalizes a newly created object.
```

Comment:
"instance method" you mean "constructor"?

Reply:
Yes fixed this in a newer commit :)